onnx/defs/operator_sets.h

```diff
@@ -965,4 +970,56 @@ inline void RegisterOnnxOperatorSetSchema() {
   RegisterOpSetSchema<OpSet_Onnx_ver14>();
 }
 
+inline void RegisterOnnxOperatorSetSchema(int target_version) {
+  // if the given version has been set, simply return
+  if (OpSchemaRegistry::GetSchemaVersion() == target_version) {
+    return;
+  }
+  OpSchemaRegistry::SetSchemaVersion(target_version);
```

Comment:
Check here to prevent repeatedly schema registration for same opset_version. If another opset_version is given, clear the previous one and load again.